css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: digital;
+  src: url(fonts/digital-7.ttf);
+}
+
+
+
+body {
+    background: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+input {
+    background-color: lightgray;
+    height: 50px;
+    width: 700px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+    border-radius: 15px;
+    
+    font: 60px digital;
+    color: darkblue;
+    padding-left: 20px;
+}
+
+#keyboard {
+    
+    position: absolute;
+    background-color: darkgray;
+    top: 200px;
+    width: 350px;
+    height: 200px;
+
+    /* position: relative; */
```

Comment:
avoid commented code in commits

Reply:
If I got you right, I need to delete all the comments in the files before committing them, right?